pkg/core/interop/context.go

```diff
@@ -140,6 +140,7 @@ func NewContractMD(name string, id int32) *ContractMD {
 	// Therefore values are taken from C# node.
 	c.NEF.Header.Compiler = "neo-core-v3.0"
 	c.NEF.Header.Magic = nef.Magic
+	c.NEF.Tokens = []nef.MethodToken{} // avoid `nil` result during JSON marshalling
```

Comment:
Somewhat scary, I'm not sure how it affects our NEF-related code.

Reply:
I'll check, although it shouldn't be critical because we're mostly concerned about serialized NEF (in contract states, e.g.), and serialized form remains the same with this change.

Reply:
So after the review:

* Native states are not affected (they don't have tokens and don't work with `CALLT`), serialized state remains the same, because empty slice and nil slice are being serialized the same way.
* Non-native states are not affected, because `NewContractMD(...)` is used for native contracts only.
* Test for non-native contracts is added to ensure that `nef.Tokens` is always not nil (we need it to not to be nil to avoid panic [here](https://github.com/nspcc-dev/neo-go/blob/master/pkg/core/interop/contract/call.go#L31)).